src/main/java/at/hannibal2/skyhanni/data/jsonobjects/repo/NEUPetsJson.kt

```diff
@@ -0,0 +1,13 @@
+package at.hannibal2.skyhanni.data.jsonobjects.repo
+
+import at.hannibal2.skyhanni.utils.NEUInternalName
+import com.google.gson.JsonObject
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+data class NEUPetsJson(
+    @Expose @SerializedName("pet_levels") val petLevels: List<Int>,
+    @Expose @SerializedName("custom_pet_leveling") val customPetLeveling: JsonObject,
```

Comment:
dont use JsonObject

Reply:
see [this](https://canary.discord.com/channels/997079228510117908/1094190239532208228/1298330284764434502)